examples/javascript/main.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Showing an example of scripting (javascript) from command line
+ *
+ * @author      Emmanuel Baccelli <emmanuel.baccelli@inria.fr>
+ *
+ * @}
+ */
+
+#include <string.h>
+#include "shell.h"
+#include "jerryscript.h"
+
+int shell_script(int argc, char **argv)
+{
+    if (argc < 2) {
+        puts("Usage: script <your script here!>\n\nTry for instance: script print('Hello World');");
+        return -1;
+    }
+
+    int count = argc - 1;
+    jerry_char_t script[(2 * SHELL_DEFAULT_BUFSIZE + 1)];
+    strcpy((char *) script, argv[1]);
+    while (count > 1) {
```

Comment:
how about ```for(int i = 1; i < argc; i++){``` instead of lines 32, 35 and 36?
If you make "script" empty with ```*script='\0'```, you can start with 0 and drop line 34 also.